### ISSUE TITLE
Draft Neighborhood Centers Content

### ISSUE THREAD

**Issue by LauraTrujillo** [cityofaustin/techstack#1619]
- [x] [Free health services at neighborhood centers](https://docs.google.com/document/d/1v-1HhTp3lhSfHb88-0DMfRR5XyVgD4jEh-2NZVP6juk/edit#)
- [x] [Social services at neighborhood centers](https://docs.google.com/document/d/18ypjn59u1J2nQnVi5GH-7SWVIIbF0gPta9-P4t-nDco/edit)
- [x] [Get information and referrals at neighborhood centers](https://docs.google.com/document/d/1a9iaGEySGXIIGvjOjipKgFyt9f_0sPJGecpS7bS3-cs/edit)
- [x] [Free child safety seats](https://docs.google.com/document/d/13OWyT96U2B3Ur1IixwlPsSY3OxpG8jCTYmee-N6fHY0/edit#)
- [x] [Basic services at neighborhood centers](https://docs.google.com/document/d/13EVa-ThRpcZaRJovt-APmnyMpPAbvVsk-8nbEWwwmh4/edit)

**Comment by LauraTrujillo** [comment]
Bring Andrew into this project

**Comment by LauraTrujillo** [comment]
draft lives here: https://docs.google.com/document/d/1Ria-t7kV_2z-d52ZZb9tIvDJ4zFCg4OKhTTVoPb7DOM/edit?usp=sharing
On to @JeffCopy for first review

**Comment by JeffCopy** [comment]
onward!

**Comment by LauraTrujillo** [comment]
@annieszot on to you to review, please :) 
If you can give me your thoughts by end of week (June 21st) I would appreciate it!

**Comment by LauraTrujillo** [comment]
Going to revamp this, per content pin-up, to focus on the services offered. Week of 6/24 I will call neighborhood centers to see who offers what.

**Comment by LauraTrujillo** [comment]
My tabs (closed them for Funshop)
https://docs.google.com/document/d/1FGp1fceqe8JOJuTjuqRDtkqiP3XArJkDo68osTu8nlk/edit
https://docs.google.com/document/d/1zY3seU2FAuZ5deFqZZtQh6_C14GQXWndyOwSgLOV6VE/edit#
http://www.austintexas.gov/neighborhood-centers
https://austinstrategicplan.bloomfire.com/posts/3290827-strategic-plan-flyers

**Comment by LauraTrujillo** [comment]
@annieszot Can you review this round of pages? I created an info page for each group of services instead of one service page for all of them. Review by Wednesday, **7/10** pretty please :)

**Comment by annieszot** [comment]
@LauraTrujillo Reviewed! Suggested edits and one question.

**Comment by LauraTrujillo** [comment]
@annieszot This issue was unclear, and I'm sorry. That was the old page- I broke them up by the types of services available, not just the centers that offer them. The drafts are listed out at the very top of this issue.

**Comment by annieszot** [comment]
Dah my mistake! On it @LauraTrujillo

**Comment by LauraTrujillo** [comment]
@annieszot nope, it was my bad!

**Comment by annieszot** [comment]
Should "Neighborhood Centers" be its own topic under Healthcare and Prevention/Housing Utilities (sometimes) topic collection? IA thoughts

@LauraTrujillo Ready for you! I edited for consistency across all info pages (titles, descriptions), and I took out "around Austin" assuming residents will know the neighborhood centers are in Austin. Open to debate, and I'm happy to give another look since the edits.
